formatter_test.go

```diff
@@ -259,3 +260,46 @@ func TestCycle(t *testing.T) {
 	*iv = *i
 	t.Logf("Example long interface cycle:\n%# v", Formatter(i))
 }
+
+func Test(t *testing.T) {
+	date := time.Date(2006, 1, 2, 3, 4, 5, .678901e9, time.Local)
+
+	s := fmt.Sprintf("%s", Formatter(date))
```

Comment:
Formatter ignores format verbs other than v with # and ' ' (space) flags. This format string needs to use `% #v` instead of `%s`.